python/test_file_generator.py

```diff
@@ -33,14 +37,10 @@ def __init__(self, ts_schema, lar_schema):
 		self.ts_field_names = list(ts_schema.field)
 
 		#load CBSA data for geography testing edits
-		use_cols = ['name', 'metDivName', 'countyFips', 'geoIdMsa', 
-		'metDivFp', 'smallCounty', 'tracts', 'stateCode']
-		
-		cbsa_cols = ['name', 'metDivName', 'state', 'countyFips', 'county', 
-		'tracts','geoIdMsa', 'metDivFp', 'smallCounty', 
-			 'stateCode', 'tractDecimal']
+		use_cols = geographic['tract_file_columns_used']
+		cbsa_cols = geographic['tract_file_columns']
```

Comment:
Can we keep CBSA in the reference line for loading the data? It's a helpful indicator for me of what the data is.